web/src/components/JSMpegPlayer.jsx

```diff
@@ -7,6 +7,20 @@ export default function JSMpegPlayer({ camera, width, height }) {
   const playerRef = useRef();
   const url = `${baseUrl.replace(/^http/, 'ws')}/live/${camera}`
 
+  let wakeLock = null;
```

Comment:
Hi @thburch, I would suggest you move lines 10 till 23 inside useEffect(). On each re-render `wakeLock` will be set to null, however, useEffect() callback will be called when the dependency list changes. In this case when the `url` get changed.

Reply:
Done. Thank you.